tools/RAiDER/losreader.py

```diff
@@ -685,17 +686,20 @@ def getTopOfAtmosphere(xyz, look_vecs, toaheight, factor=None):
     return pos
 
 
-def get_orbit(orbit_file, ref_time, pad=600): 
+def get_orbit(orbit_file, ref_time, pad):
```

Comment:
Is there a reason not to set a default? Seems like this function should have one.

Reply:
The pad argument is passed several times throughout the code (i think 3) through higher level functions. This one is relatively low down so a higher one sets the padding. When debugging, I found it difficult to track where actually the padding value is getting set. So I set to just have one default value that's used at the top. I'm not sure what the best practice is but this seemed easiest for me.